docs/src/screens/home/features.js

```diff
@@ -6,41 +6,88 @@ import { BodyCopy } from '../../components/body-copy';
 import { SecondaryTitle } from '../../components/secondary-title';
 import { SectionTitle } from '../../components/section-title';
 import { Wrapper } from '../../components/wrapper';
+import { Stack } from '../../components/stack';
 import { theme } from '../../themes/theme';
 
-const FeatureCard = styled.div`
-  margin: 0 0 4rem;
-  width: 100%;
-  @media (min-width: 768px) {
-    margin: 0;
-    width: calc(1 / 3 * 100% - (1 - 1 / 3) * 40px);
+const FeaturesContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  max-width: 75%;
+  align-self: center;
+
+  > * {
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  > * + * {
+    margin-top: 1rem;
   }
-  @media (min-width: 1024px) {
-    width: calc(1 / 3 * 100% - (1 - 1 / 3) * 80px);
+
+  @media ${p => p.theme.media.sm} {
+    max-width: none;
+    flex-direction: row;
+
+    > * {
+      margin-top: 0;
```

Comment:
Is there an advantage to explicitly setting every margin property vs `margin: 0`? Does this allow for more explicit override?

Reply:
Good shout! Yep you got it, the main reason is to allow your child elements to maintain any sort of left or right margin that they may be getting from another class. With this stack layout you really want to enforce vertical spacing without affecting the horizontal. More useful when it comes to an abstracted mixin in Sass – doesn't matter as much here when all classes this selector is being applied to are already in the same file.